### PR TITLE
datatable style fix

### PR DESCRIPTION
**Pull request by odb23**
issue #17 fix and datatable style update

- Reduced the size of the pagination component
- Added padding around datatable entries, search, info and pagination.

### CODE REVIEW

File: src/components/DataTable/DataTable.tsx [hunk 1]
```diff
@@ -472,7 +472,12 @@ const DataTable = (props: Props) => {
   return (
     <ThemeProvider theme={theme}>
       <div data-test='datatable' className={tableClasses}>
-        <div className={'row'}>
+        <div style={{
+          display: "flex",
+          flexWrap: "wrap",
+          justifyContent: "space-between",
```
Comment (Speedwares) [inline src/components/DataTable/DataTable.tsx]: Is there any reason why you chose inline styling here?

File: src/components/DataTable/DataTableSelect/DataTableSelect.tsx [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
+import { ThemeProvider } from "styled-components";
+import { theme } from "../../../theme";
 
 interface Props {
-  entries?: any
-  onChange?: any
-  value?: number
-  label?: any
+  entries?: any;
+  onChange?: any;
```
Comment (Speedwares) [inline src/components/DataTable/DataTableSelect/DataTableSelect.tsx]: Can we remove the semicolons

File: src/components/DataTable/DataTableSelect/DataTableSelect.tsx [hunk 3]
```diff
@@ -1,55 +1,61 @@
...
   onChange: PropTypes.func.isRequired,
-  value: PropTypes.number.isRequired
-}
+  value: PropTypes.number.isRequired,
+};
 
-export default DataTableSelect
-export { DataTableSelect as MDBDataTableSelect }
+export default DataTableSelect;
+export { DataTableSelect as MDBDataTableSelect };
```
Comment (Speedwares) [inline src/components/DataTable/DataTableSelect/DataTableSelect.tsx]: Same here as well

File: src/components/DataTable/DataTable.tsx (hunk 1 above)
Comment (odb23) [reply src/components/DataTable/DataTable.tsx]: No specific reason. I will make a change to this.

File: src/components/DataTable/DataTableSelect/DataTableSelect.tsx (hunk 2 above)
Comment (odb23) [reply src/components/DataTable/DataTableSelect/DataTableSelect.tsx]: Definitely...

File: src/components/DataTable/DataTableSelect/DataTableSelect.tsx (hunk 3 above)
Comment (odb23) [reply src/components/DataTable/DataTableSelect/DataTableSelect.tsx]: Ok. will change this too

**Comment by odb23** [comment]
@Speedwares, Kindly review the new changes I made.

**Review by Speedwares — APPROVED** [review]

**Pull request merged**